### DISCUSSION TITLE
Chat/Messageing component

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#1052] [Ideas]
I really like the chat box and so do our users. There are just some issues with it that I believe I had mentioned in discussion at one point or another, that were addressed. I would really like this feature to be functional in wave apps. 

It would also be cool to associate potentially some sort of messaging capability that could piggy back off the persona cards maybe? 

Totally just a thought, and I am sure there are caveats. Direct Messaging within the wave apps would be an awesome feature!

**Comment by mturoci** [comment]
cc @lo5
